### DISCUSSION TITLE
マップ上表示、ステータス表示を前のままで維持したい

### DISCUSSION THREAD

**Discussion by aera-ngmansion** [ngmansion/FE8N#51] [Q&A] [answered]
Aeraさん、マップ上表示、ステータス表示を前のままで維持したいので添付ファイルのものに差し替えて頂けないでしょうか？
よろしくお願いします。
Engine Hacks→Ｎ→

_Originally posted by @Isaac45132 in https://github.com/ngmansion/FE8N/issues/46#issuecomment-739504287_

**Comment by aera-ngmansion** [comment]
すぐに対応できるとは言えませんが、添付ファイルとはどこにあるでしょうか

また、利用法の話になりますが、
基本的に、別の話題のissueに対して関係ないコメントをするのはお控えいただき、
今回のように新しいissueを作成してコメントいただくようお願いします

**Comment by Isaac45132** [comment]
了解しました。
メールアドレスのほうにファイルを送付させて頂きます。

**Comment by aera-ngmansion** [comment]
ご回答ありがとうございます

すみませんが、今メールアドレスを使えないので、別の方法でお願いします

**Comment by Isaac45132** [comment]
了解しました。下記あぷろだの上から２つをＤＬお願いします。https://ux.getuploader.com/AZA1955/


     ----- Original Message -----

**Comment by aera-ngmansion** [comment]
ファイル確認しました。
確認したうえで改めて質問があるのですが、これは私に何を依頼なさっているのでしょうか？
何をしたいのか？(= 前のままとは何を指す？)と、このファイルをどうすればいいのか？の２つが分かりませんでした。
お教え願えないでしょうか。

**Comment by Isaac45132** [comment]
お渡ししたファイルをTNHのgithubの該当フォルダと差し替えて頂けないでしょうか？そして、この部分はこれ以上変更無しでお願いしたいとおもっています。
_N→EquipmentPopup_N→status2

     ----- Original Message -----

**Comment by aera-ngmansion** [comment, marked as answer]
意図が分からないのと、今後の事に絶対の約束はしかねるという事はありますが、
何を実施するかは分かったので、そのままの事は行いました。

**Comment by Isaac45132** [comment]
すいません、対応ありがとうございました！

     ----- Original Message -----
